cloudinit/sources/helpers/vultr.py

```diff
@@ -114,10 +119,11 @@ def generate_network_config(interfaces):
         public = generate_public_network_interface(interfaces[0])
         network['config'].append(public)
 
-    # Prepare interface 1, private
+    # Prepare additional interfaces, private
     if len(interfaces) > 1:
-        private = generate_private_network_interface(interfaces[1])
-        network['config'].append(private)
+        for i in range(1, len(interfaces)):
+            private = generate_private_network_interface(interfaces[i])
+            network['config'].append(private)
```

Comment:
Could this be simplified to the following to avoid extra indents?
```suggestion
for i in range(1, len(interfaces)):
    private = generate_private_network_interface(interfaces[i])
    network['config'].append(private)
```

Reply:
Yeah, that if was superfluous here, removed.